src/main/java/dev/aziz/movies/entities/Actor.java

```diff
@@ -29,4 +33,12 @@ public class Actor {
 
     @NotBlank(message = "Lastname should not be empty")
     private String lastName;
+
+    @ManyToMany
+    @JoinTable(
+            name = "actor_genre",
+            joinColumns = @JoinColumn(name = "actor_id"),
+            inverseJoinColumns = @JoinColumn(name = "genre_id")
+    )
+    private List<Genre> genres;
```

Comment:
You don't need this new table. You can get the genres of an actor or director by looking for the movies.